src/doc/reference/glossary/commutative_operation.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Commutative operation
+tags: commutative operation
+---
+An operation (op), over a type (T), is
+commutative if a op b = b op a for
+any two objects, a and b, of type T.
```

Comment:
Should the mathematical symbols here be formatted differently, to distinguish them from ordinary text?  I think that would look better, but I don't know what formatting we use for math.

Reply:
I converted to LaTeX / mathjax.